examples/hello-world-python/helloworld.log

```diff
@@ -0,0 +1,11 @@
+Read yaml helloworld.yaml
```

Comment:
I would avoid putting the Shroud generated files into the repository. Every time Shroud is changed it may be necessary to update them.  I'll look at adding a top level Make target to test the examples to make sure they still work - ``make test-examples``.
There is already a ``make test-all`` which compiles the reference files.

Reply:
Removed. I thought they might be useful for reference, but you're right they will change.